Formula/gcal.rb

```diff
@@ -28,8 +28,13 @@ class Gcal < Formula
   end
 
   def install
-    # @setshortcontentsaftertitlepage was removed in Texinfo 6.3
-    inreplace "doc/en/gcal.texi", "@setshortcontentsaftertitlepage\n", "" if OS.linux?
+    if OS.linux? || MacOS.version >= :ventura
+      # @setshortcontentsaftertitlepage was removed in Texinfo 6.3
+      inreplace "doc/en/gcal.texi", "@setshortcontentsaftertitlepage\n", ""
+
+      # Workaround for Xcode 14.3.
+      ENV.append_to_cflags "-Wno-implicit-function-declaration"
```

Comment:
Set this after calling `configure` to avoid breaking function detection.

Reply:
Weird, thought it worked for me locally like that ... let's try again then